students/jungmin/westagram/users/views.py

```diff
@@ -47,9 +48,19 @@ def post(self, request):
             email    = data['email']
             password = data['password']
 
-            if not User.objects.filter(email=email, password=password).exists():
+            user = User.objects.get(email=email)
+
+            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
                 return JsonResponse({'message':'INVALID_USER'}, status=401)
-            
-            return JsonResponse({'message':'SUCCESS'}, status=200)
+
+            return JsonResponse(
+                {
+                    'message':'SUCCESS', 
+                    'token': jwt.encode({'user_id': user.id}, SECRET_KEY, ALGORITHM)
```

Comment:
```suggestion
                    'token': jwt.encode({'user_id': user.id}, settings.SECRET_KEY, settings.ALGORITHM)
```
위의 리뷰에 따라 수정한다면 다음과 같이 바꿀 수 있습니다.